content/courses/onchain-development/local-setup.md

```diff
@@ -2,116 +2,131 @@
 
-- To develop onchain programs on your machine, you need **Solana CLI**, **Rust**
+- To develop onchain programs locally, you need the **Solana CLI**, **Rust**,
   and (optional, but recommended) **Anchor**.
-- You can use `anchor init` to create a new blank Anchor project
-- `anchor test` runs your tests, and also builds your code.
+- You can use `anchor init` to create a new blank Anchor project.
+- `anchor test` runs your tests and also builds your code.
 
 ## Lesson
 
-There's no lesson here! Let's install Solana CLI tools, the Rust SDK, and
-Anchor, and create a test program to ensure that our setup works.
+There's no formal lesson here! Let's get hands-on by installing the Solana CLI
+tools, the Rust SDK, and Anchor, and create a test program to verify that
+everything is set up correctly.
 
 ## Lab
 
-#### Extra steps for Windows users
+### Extra steps for Windows users
 
-Firstly install
+First, install
 [Windows Terminal](https://apps.microsoft.com/detail/9N0DX20HK701) from the
 Microsoft store.
 
-Then
+Next,
 [install Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install).
-WSL provides a Linux environment that launches instantly whenever you need it
-and doesn't slow your computer down.
+WSL provides a Linux environment that launches instantly when needed without
+slowing down your computer.
 
-Start Windows Terminal, launch an 'Ubuntu' session inside the terminal, and
-proceed with the rest of these steps.
+Open Windows Terminal, start an 'Ubuntu' session and proceed with the rest of
+these steps.
 
-#### Download Rust
+### Download Rust
 
-First, download Rust by
+First, install Rust by
 [following the instructions](https://www.rust-lang.org/tools/install):
 
-```
+```bash
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
-#### Download the Solana CLI tools
+### Download the Solana CLI tools
 
-Next
-[download the Solana CLI tools](https://docs.solana.com/cli/install-solana-cli-tools).
+Next,
+[download the Solana CLI tools](https://docs.solana.com/cli/install-solana-cli-tools):
 
-```
+```bash
 sh -c "$(curl -sSfL https://release.anza.xyz/stable/install)"
 ```
 
-Afterwards, `solana -V` should show `solana-cli 1.18.x` (any number for `x` is
-fine).
+After installation, `solana -V` should display `solana-cli 1.18.x` (where `x`
+can be any number).
 
-#### Download Anchor
+### Running the Solana Test Validator
 
-Finally [download Anchor](https://www.anchor-lang.com/docs/installation):
+The Solana Test Validator is a local emulator for the Solana blockchain. It
+provides developers with a private and controlled environment to build and test
+Solana programs without needing to connect to a public testnet or mainnet.
 
+To start the Solana Test Validator, run the following command:
+
+```bash
+solana-test-validator
 ```
+
+For more detailed information, you can refer to the
+[Solana Test Validator guide](https://solana.com/developers/guides/getstarted/solana-test-validator).
+
```

Comment:
Show people what success looks looks like (ie, what the validator looks like) then ask them to press Ctrl C to cancel it, since we need `anchor init` to run later.

Reply:
Sure adding it.

Reply:
Updated output of validator and how to cancel it.